examples/alife-boid/src/vector.rs

```diff
@@ -53,7 +48,15 @@ impl Vector {
         if size == 0.0 {
             return;
         }
-        self.div(size)
+        *self /= size;
+    }
+
+    pub fn size_square(&mut self) {
```

Comment:
That's not quite what I meant.
I was trying to point out that you're needlessly squaring the square root like this: `sqrt(x^2 + y^2)^2`.
Instead of squaring the `.size()` you should use a new method `.size_square()` which returns `x^2 + y^2` so you can skip taking the square root.